abd-clam/src/cakes/knn/expanding_threshold.rs

```diff
@@ -106,7 +106,7 @@ fn leaf_into_hits<T: Send + Sync + Copy, U: Number, D: Dataset<T, U>>(
     } else {
         tree.data().query_to_many(query, is)
     };
-    is.iter().zip(ds.into_iter()).for_each(|(&i, d)| {
```

Comment:
I'd love a clearer explanation of which 1.72 change obviated the need to consume the Iterable. Or was this just extraneous all along?